src/components/FavouritesMoviesComponent/FavouritesMoviesComponent.tsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import "./style.css";
+import { useHistory } from "react-router-dom";
+import { useAppDispatch } from "../../hooks/useAppDispatch";
+import { useAppSelector } from "../../hooks/useAppSelector";
+import { Routes } from "../../constants/Routes";
+import { deleteChekedMovie } from "../../store/reducers/favouriteSlice";
+import { Close } from "@mui/icons-material";
+import ImageWithOnError from "../ImageWithOnError.tsx/ImageWithOnError";
+
+const FavouritesMoviesComponent = () => {
+  const router = useHistory();
+  const dispatch = useAppDispatch();
+  const { favouriteMovie } = useAppSelector(
+    (state) => state.movieFavouriteReducer
+  );
+  return (
+    <>
+      {favouriteMovie.length ? (
+        favouriteMovie.map((el: any) => (
```

Comment:
el should have types since it's element of array. If not, add type for favouriteMovies array in redux